applications/ContactStructuralMechanicsApplication/custom_strategies/custom_strategies/residualbased_newton_raphson_contact_strategy.h

```diff
@@ -30,7 +30,7 @@
 #include "utilities/variable_utils.h"
 #include "utilities/color_utilities.h"
 #include "utilities/math_utils.h"
-#include "custom_utilities/process_factory_utility.h"
+#include "utilities/process_factory_utility.h"
```

Comment:
this is of the core?

Reply:
Must be moved in order to use it inn the Meshing